src/user-guide/running-workflows/workflow-run-modes.rst

```diff
@@ -0,0 +1,113 @@
+.. _workflow-run-modes:
+
+Workflow Run Modes
+------------------
+
+Cylc can run a workflow without running the real jobs, which may be
+useful when developing workflows. Tasks submitted
+using :ref:`workflow-run-modes.dummy` and
+:ref:`workflow-run-modes.simulation` can be used to understand how different outcomes
```

Comment:
Sorry if I'm being overly picky, but to my mind tasks don't get "submitted" in simulation mode, because submission is the process of submitting a job to run on the system. I suppose we could spin it that way, but IMO better to avoid even raising the question in user's minds.

```suggestion
useful when developing workflows. Tasks 
in :ref:`workflow-run-modes.dummy` and
:ref:`workflow-run-modes.simulation` can help you to understand how different outcomes
```

Reply:
If you want to be picky you could say that tasks submit jobs and are not themselves submitted. 

I have accepted this, though I'm not sure I agree. I'd describe skip and simulation jobs as being submitted to the scheduler.

Reply:
> If you want to be picky you could say that tasks submit jobs and are not themselves submitted.

Correct, we shouldn't say that tasks get submitted, just being lazy. They have a submitted state, which means they submitted a job.

> I'd describe skip and simulation jobs as being submitted to the scheduler.

Yeah, that's what I meant by "I suppose we could spin it that way".

But I think it's easy for users to grok if we just say jobs get submitted.